### ISSUE TITLE
Feature: disable/enable LoginItems

### ISSUE THREAD

**Issue by hazcod** [hazcod/maclaunch#27]
We currently still need to support to enable/disable login items in Ventura.
This would work by:

1. Adding it in /var/db/com.apple.xpc.launchd/disabled.*.plist with `<key>NAME</key>\n<true/>` to disable.
2. Adding it in /var/db/com.apple.xpc.launchd/disabled.*.plist with `<key>NAME</key>\n<false/>` to disable.

**Comment by masonc15** [comment]
I think this is the issue I'm running into when trying to disable a startup item:

```shell
$ maclaunch disable com.wiheads.paste-setapp-LaunchAtLoginHelper
/opt/homebrew/bin/maclaunch: line 608: disableLoginItems: command not found
```

Just wanted to add it for context.  Thanks!

**Comment by ctsstc** [comment]
I noticed that `disableLoginItems` was added in the following commit, but there hasn't been an accompanying function for it. I searched google for the command/function and nothing shows up.
https://github.com/hazcod/maclaunch/commit/eb91b579651f419652430db159fabcbe9476ab19

The function left unimplemented so that it would throw an error until this is implemented thus fixing it?

**Comment by hazcod** [comment]
Yes, indeed. This issue is specifically about implementing that function.
